### DISCUSSION TITLE
Распаковка библиотеки ESPAsyncWebServer.zip

### DISCUSSION THREAD

**Discussion by Mihger** [vvip-68/LedPanelWiFi#111] [Общие вопросы]
Архив с проектом скачивал полностью несколько раз. Он исправно распаковывается. А проблема с распаковкой библиотеки так и не пропала(писал ранее о проблеме), пробовал на win7 и win10, разными архиваторами. Проблема возникает только с библиотеками в архивах находящиеся в папке "для esp8266 core 3.1.2 FastLed 3.6.0"
![image](https://github.com/user-attachments/assets/68a3eb12-478c-40dc-9a32-e17871004e0a)
![image](https://github.com/user-attachments/assets/69c3bc93-d55d-4573-aba0-b4435f104762)
![image](https://github.com/user-attachments/assets/723fa28f-6c5f-4593-b100-a45eef197997)
Второй архив также не удается распаковать 
![image](https://github.com/user-attachments/assets/3d77a497-1e99-455d-9996-509e99d48316)
![image](https://github.com/user-attachments/assets/ba1a47f6-ed18-4fdc-9066-bcb8d04cbb55)
![image](https://github.com/user-attachments/assets/325cbdc7-9890-46d4-8ff6-9227176d4512)
Ошибка при распаковке на win10 через встроенный архиватор
![image](https://github.com/user-attachments/assets/309e8dd5-b2ed-40f8-a734-2a22416326a4)

**Comment by vvip-68** [comment]
Проблема в том, что эта проблема только у вас.
Проект повторили к этому новому году десяток человек, и ни у кого проблемы с распаковкой этих архивов не возникло.
Я не знаю чем вам помочь.

**Reply by vvip-68** [reply]
Попробуйте скачать с официального сайта архиватор и установить его
https://www.winzip.com/en/download/winzip/
Версия платная - но 21 день - пробный период. Вам хватит, чтобы распаковать архив.
Или найдите где-нибудь ломанную / бесплатную версию архиватора.

Или этот 
https://www.win-rar.com/start.html?&L=4

Или обновите наконец ваш WinRar архиватор до свежей версии - возможна старая версия не понимает этого формата сжатия

**Reply by vvip-68** [reply]
Проверено на WinRar вот этой версии. Архив распаковывает без проблем.
![image](https://github.com/user-attachments/assets/6488e2c7-ca2c-4c7a-bb4e-2b410e3c04a2)

**Reply by Mihger** [reply]
Хорошо, спасибо за помощь! Буду искать проблемы у себя) С наступающим Новым годом!!!)
